### PR TITLE
Remove Fugu reports that require a user gesture

### PR DESCRIPTION
**Pull request by tomayac**
Also in response to #221.

### CODE REVIEW

**Comment by tomayac** [comment]
Related PR: https://github.com/HTTPArchive/bigquery/pull/88.

File: config/reports.json [hunk 1]
```diff
@@ -1317,38 +1317,22 @@
...
-			"webNFCRead",
-			"webNFCWrite",
-			"webSerial",
-			"nativeFileSystem",
 			"contentIndex",
 			"quicTransport",
 			"idleDetection",
-			"webSocketStream",
-			"shapeDetectionFace",
-			"shapeDetectionText"
```
Comment (rviscomi) [inline config/reports.json:1351]: If these features can never be detected in HA, should we remove their metric definitions above and corresponding SQL files in https://github.com/HTTPArchive/bigquery/pull/88?

Comment (tomayac) [reply config/reports.json:1351]: If the collection methodology changes, could this table contain data in the future?

Comment (rviscomi) [reply config/reports.json:1351]: Theoretically, but I don't foresee HA simulating user gestures anytime soon.

Comment (tomayac) [reply config/reports.json:1351]: Let me axe them then. They're easy to re-establish should we need them in the future.

**Review by rviscomi — APPROVED** [review]

**Comment by rviscomi** [comment]
@tomayac I've staged these changes to https://httparchive-staging.ue.r.appspot.com/reports/project-fugu and I'm getting the following error: `'NoneType' object is not subscriptable`. Maybe one deleted metric wasn't removed from the report config?

**Comment by tomayac** [comment]
Ooops, sorry. I killed one too many, fixed in https://github.com/HTTPArchive/httparchive.org/commit/dec36c82981292a1acfd54ebd5b2e1e645628eab.

**Comment by rviscomi** [comment]
Thanks! Looks good in staging. Deploying to prod now.

**Comment by christianliebel** [comment]
@rviscomi Hi Rick, on prod (https://httparchive.org/reports/project-fugu) I can only see the data until August 1, 2020. Shouldn't the data from the September crawl also be there? Thanks!

**Comment by rviscomi** [comment]
Normally yes the September data should be available by now, thanks for pointing that out. I think the issue may be related to #225. Once that's resolved I'll restart the metrics pipeline (or wait for it to start again at the end of the month)

In the context of the PWA chapter of the Web Almanac, I'd recommend sourcing the data from BigQuery so we can follow a consistent data pipeline methodology for all metrics/chapters. Along those lines, you should also be looking at August 2020 data for parity with other chapters.

**Pull request merged**